### PR TITLE
 ERMS-2013, ERMS-2113

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: app/grails-app/conf/Config.groovy [hunk 1]
```diff
@@ -435,7 +435,7 @@ grails.project.dependency.resolver = "maven"
 environments {
     development {
         grails.logging.jul.usebridge = true
-        grails.serverURL = "http://localhost:8080/laser"
+        grails.serverURL = "http://localhost:8090/laser"
```
Comment (croqueGrec09) [inline app/grails-app/conf/Config.groovy]: Why that?

File: app/grails-app/views/title/list.gsp [hunk 2]
```diff
@@ -25,7 +25,7 @@
           </div>
           <div class="field">
             <label for="filter">${message(code: 'title.search_in')}</label>
-            <g:select class="ui dropdown" id="filter" name="filter" from="${[[key:'title',value:"${message(code: 'title.title.label')}"],[key:'publisher',value:"${message(code:'title.publisher.label')}"],[key:'',value:"${message(code: 'title.all.label')}"]]}" optionKey="key" optionValue="value" value="${params.filter}"/>
+            <g:select class="ui dropdown" id="filter" name="filter" from="${[[key:'name',value:"${message(code: 'title.title.label')}"],[key:'publisher',value:"${message(code:'title.publisher.label')}"],[key:'',value:"${message(code: 'title.all.label')}"]]}" optionKey="key" optionValue="value" value="${params.filter}"/>
```
Comment (croqueGrec09) [inline app/grails-app/views/title/list.gsp]: Sure? Name property for TitleInstance is title

Comment (Moetez16) [reply app/grails-app/views/title/list.gsp]: yes

Comment (croqueGrec09) [reply app/grails-app/views/title/list.gsp]: Alright ...

**Pull request merged**